python/ray/remote_function.py

```diff
@@ -169,6 +170,7 @@ def remote(self, *args, **kwargs):
 
         return FuncWrapper()
 
+    @_tracing_task_invocation
```

Comment:
Can we not turn this on by default now?

Reply:
Same for other parts

Reply:
it's not on by default, guarded by a env var feature flag